jenkins/scripts/node-test-commit-diagnostics.sh

```diff
@@ -39,4 +39,11 @@ mv ${DIAGFILE} ${DIAGFILE}-OLD || true
 tail -c 20000000 ${DIAGFILE}-OLD > ${DIAGFILE} || true
 rm ${DIAGFILE}-OLD || true
 set -x
-pgrep node || true
+perl -MFile::Basename -e '
+  my @r = ();
+  for (qx[ps ax]) {
+    my @F = split;
+    push @r, $F[0] if $F[4] && basename($F[4]) eq "node"
+  }
+  print "@r\n" if @r
+' || true
```

Comment:
is the `|| true` needed anymore? pgrep would fail if the process isn't found, perl will just list what it found and exit with sucess.

Reply:
Removed